pkg/client/client.go

```diff
@@ -271,7 +271,7 @@ var (
 // FormatStatusResponse writes a StatusResponse as a string to the writer. The bit mask sd controls
 // whether a additional details are printed about a certain aspect of the status. In case there are
 // errors, some details may be printed regardless of the value of sd.
-func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetails) {
+func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetails, ne bool) {
```

Comment:
Nit: Please make the variable name more descriptive.
For example:
```suggestion
func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetails, encryptStatus bool) {
```
Please also change invocations of this function to have the variable passed be something like "encryptStatus" as well.